src/offline/cable_write.F90

```diff
@@ -901,13 +883,6 @@ SUBROUTINE write_output_variable_r1(ktau, ncid, varID, vname, var_r1,        &
                 ! patches):
                 otmp3xyt(land_x(i), land_y(i), 1) = SUM(var_r1(landpt(i)%cstart:   &
                      landpt(i)%cend) * patch(landpt(i)%cstart:landpt(i)%cend)%frac)
```

Comment:
@ccarouge does processing like this affect check range bounds for variables like `HSoil`, where it's just not the index anymore

Reply:
nvm it doesn't